frontend/public/components/masthead-toolbar.jsx

```diff
@@ -572,6 +573,7 @@ class MastheadToolbarContents_ extends React.Component {
                 <PlusCircleIcon className="co-masthead-icon" />
               </Button>
             </ToolbarItem>
+            <CloudShellMastheadButton />
```

Comment:
Let's comment this out before merging because we need to fix the workspace namespace and resource name before it can be used.

Reply:
commented out